spec/eventually_spec.rb

```diff
@@ -9,4 +9,14 @@
     eventually { @test = outcome }
     expect(@test).to eq outcome
   end
+
+  it 'can attach to expect statements' do
+    @test = ->{
+      responses = %w(duck duck goose goose).cycle
+      ->{ responses.next }
+    }.call
+    outcome = 'goose'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.